expo/tests/gbe/test_assign_volunteer.py

```diff
@@ -133,17 +139,20 @@ def test_assign_volunteer_show_current_events(self):
 
```

Comment:
Hmmm... I see the bug I found isn't really represented by any time related test case.
A nice to have would be a test that breaks if the time window isn't there (when it should be).

But I'm not hung up on it... once the format is working correctly, I'm not overly worried about it.